components/raftstore/src/coprocessor/split_check/keys.rs

```diff
@@ -45,7 +48,9 @@ where
     E: KvEngine,
 {
     fn on_kv(&mut self, _: &mut ObserverContext<'_>, key: &KeyEntry) -> bool {
-        if !key.is_commit_version() {
+        // For txn, only check keys difference in write cf.
+        // For rawkv, write cf should be empty so only default cf will be checked.
+        if self.cf == CF_WRITE && !key.is_commit_version() {
```

Comment:
What if a txnkv has an empty key for `CF_WRITE` but has some keys for `CF_DEFAULT`?
I think the behavior is different from the one before. Is it has some side effect?